src/js/tech/middleware.js

```diff
@@ -2,6 +2,8 @@ import { assign } from '../utils/obj.js';
 
 const middlewares = {};
 
+export const TERMINATOR = 'TERMINATOR';
```

Comment:
we should make this be an object instance rather than a string since objects only equal themselves. Then the string will also be a valid value.

Reply:
what do you mean by "the string will also be a valid value"

Reply:
Did you mean change this to `export const TERMINATOR = () => { return 'TERMINATOR'; }` and then here:

https://github.com/ldayananda/video.js/blob/52011c3bc175897e735ec5018159de222db0980c/src/js/video.js#L752

change it to be `videojs.TERMINATOR = TERMINATOR()` ?

Reply:
nope, literally just `TERMINATOR = {}`

Reply:
Oh, you meant "the string will also be a valid value" to mean that an allowed method could return the string 'TERMINATOR'! 👍  will do